mava/systems/jax/config_test.py

```diff
@@ -149,6 +149,28 @@ def test_update_config(
     assert not hasattr(config, "setting")
 
 
+def test_update_config_twice(
```

Comment:
so will the interplay between component config and the system be to specify your configs with this class which is then used to create actual components in the builder?

Reply:
Looking forward to seeing a run class with everything together then doing a tf side by side

Reply:
The config class is for automatically generating the default configs for the system based on the initial components. But the idea is that anyone can then add new components (for which the config will be handled as well by the config class), or the parameters can be changed for different components via the `set_parameters. I hope this answers your question? :)

Reply:
Yes, thanks